src/main/java/org/jenkinsci/plugins/builduser/varsetter/impl/TimerTriggerCauseDeterminant.java

```diff
@@ -0,0 +1,30 @@
+package org.jenkinsci.plugins.builduser.varsetter.impl;
+
+import java.util.Map;
+
+import org.jenkinsci.plugins.builduser.utils.UsernameUtils;
+import org.jenkinsci.plugins.builduser.varsetter.IUsernameSettable;
+
+import hudson.triggers.TimerTrigger;
+
+public class TimerTriggerCauseDeterminant implements IUsernameSettable<TimerTrigger.TimerTriggerCause> {
+
+	private static final String TIMER_TRIGGER_DUMMY_USER_NAME = "Timer Trigger";
+	private static final String TIMER_TRIGGER_DUMMY_USER_ID = "timer";
+
+    @Override
```

Comment:
There is a mix of spaces and tabs and I wouldn't like to reformat everything. I kept the original convention - tabs.